CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v2.0.0-beta.13 [unreleased]
+
+### Features
+
+1. [18480](https://github.com/influxdata/influxdb/pull/18480): Allows tasks to open in new tabs
```

Comment:
i don't think this is a feature, it's like, table stakes of a browser.

if it has to be in the changelog, i think it should go under ` ### UI Enhancements`